main/utils/text_entry.c

```diff
@@ -25,7 +25,7 @@ typedef enum
     KEY_SHIFT     = 0x01,
     KEY_CAPSLOCK  = 0x02,
     KEY_BACKSPACE = 0x03,
-    KEY_SPACE     = 0x04,
+    KEY_SPACE     = 0x20,
```

Comment:
You know backspace is `0x10` in ASCII, but you didn't change that.

Reply:
I changed it this way because `KEY_SPACE` wasn't being handled in its own case in the `switch`, so it ended up in the default case with all the other characters and inserted a `0x04` into the text which made everything unhappy. I figured since the other control characters were working fine they didn't need to be changed.